Added maple syrup.

KathyRyals opened the pull request:
![image](https://user-images.githubusercontent.com/65850818/91656565-06601c00-eaba-11ea-8fbc-21d2965bf2cb.png)



## About The Pull Request
Adds two maple syrup bottle to the fridge in the bar.

Credit for the sprite goes to cybersaber101
![image](https://user-images.githubusercontent.com/65850818/91783429-809ab880-ec00-11ea-9e91-f350f6e3cc65.png)



## Why It's Good For The Game
More variety good.
Also for Sierra : I replaced the 0 with FALSE and the 1 with true _fingerguns_

## Changelog

:cl: Kathy Ryals & cybersaber101
rcsadd: The Torch's bar has been supplied with maple syrup.
/:cl:

PsiOmegaDelta commented on code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:
```diff
@@ -1151,6 +1151,15 @@
+	name = "Maple Syrup"
+	description = "Canada is still going at it, no one can stop them."
+	taste_description = "liquid, nutty , sugary goodness"
+	color = "#B24403"
```
"#b24403"
Travis is going to fail this for using uppercase letters

KathyRyals replied:
Thank you !

KathyRyals commented:
Squashed !

Spookerton commented on code/modules/reagents/reagent_containers/food/drinks/bottle.dm:
```diff
@@ -584,12 +584,22 @@
 
+/obj/item/weapon/reagent_containers/food/drinks/bottle/maplesyrup
+	name = "Maple Syrup"
+	desc = "Sweet, sirupy goodness."
```
```suggestion
	desc = "Sweet, syrupy goodness."
```

Spookerton commented on code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:
```diff
@@ -1151,6 +1151,15 @@
+/datum/reagent/drink/maplesyrup
+	name = "Maple Syrup"
+	description = "Canada is still going at it, no one can stop them."
+	taste_description = "liquid, nutty , sugary goodness"
```
```suggestion
	taste_description = "liquid, nutty, sugary goodness"
```

Spookerton commented on code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:
```diff
@@ -1151,6 +1151,15 @@
+	color = "#b24403"
+
+	glass_name = "maple syrup"
+	glass_desc = "Thick and very sweet, the perfect canadian treat to enjoy under a clear Sky."
```
```suggestion
	glass_desc = "Thick and very sweet, the perfect Canadian treat to enjoy under a clear sky."
```

KathyRyals commented:
Will fix !

PsiOmegaDelta commented:
Noting that the commit message is rather so-so

Albens commented:
Author says DNM

SierraKomodo commented:
MArked as draft, requested by author:

> Kathonika RyalidesToday at 12:34 PM
> `@Developers` can someone DNM my maple syrup or ?
> Someone sent me a better Sprite and I'll update it when I'm home

SierraKomodo commented on code/modules/reagents/reagent_containers/food/drinks/bottle.dm:
```diff
@@ -584,12 +584,22 @@
 	reagents.add_reagent(/datum/reagent/drink/juice/lemon, 100)
 
+/obj/item/weapon/reagent_containers/food/drinks/bottle/maplesyrup
+	name = "Maple Syrup"
```
Seems odd that this is capitalized - If it's meant to be a proper name, maybe add a brand name or something to it. Otherwise, it should be lowercase. I'd also add `bottle` to the name.

In chat references to the object look a bit odd otherwise:

![dreamseeker_9yPHqf2hG8](https://user-images.githubusercontent.com/11140088/91915991-d5761780-ec70-11ea-9703-b84615daf5c2.png)

```suggestion
	name = "maple syrup bottle"
```

SierraKomodo commented on code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:
```diff
@@ -1151,6 +1151,15 @@
+/datum/reagent/drink/maplesyrup
+	name = "Maple Syrup"
+	description = "Canada is still going at it, no one can stop them."
+	taste_description = "liquid, nutty, sugary goodness"
```
-liquid - Not really a taste description. If it's a drink, it's assumed liquid. If it's solid food that has maple syrup, why would it describe as a liquid?
```suggestion
	taste_description = "nutty, sugary goodness"
```

SierraKomodo commented on code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:
```diff
@@ -1151,6 +1151,15 @@
 	glass_name = "grenadine syrup"
 	glass_desc = "Sweet and tangy, a bar syrup used to add color or flavor to drinks."
 
+/datum/reagent/drink/maplesyrup
```
This should probably have `handle_sugar()` processing added for unathi since maple syrup is a sugary substance. Here's an example copied off `/datum/reagent/drink/juice`

NOTE: Straight sugar has an efficiency value (Third parameter of `handle_sugar()`) of 1.0, juice has a value of 0.5. Find a value that makes sense to you for maple syrup based on that.

```dm
/datum/reagent/drink/maplesyrup/affect_ingest(mob/living/carbon/human/M, alien, removed)
	..()
	if(alien == IS_UNATHI)
		var/datum/species/unathi/S = M.species
		S.handle_sugar(M, src, 0.5)
```

KathyRyals committed:
Added mapple syrup

Fixed typos, added icon.


Changed icon


Fixed an issue for Travis


Fixed Typos in descriptions


Better icon


Added handle_sugar for unathis, fixed some spelling

Shadowtail117 commented on code/modules/reagents/reagent_containers/food/drinks/bottle.dm:
```diff
@@ -584,12 +584,22 @@
+/obj/item/weapon/reagent_containers/food/drinks/bottle/maplesyrup
+	name = "maple syrup bottle"
+	desc = "Sweet, syrupy goodness."
+	icon_state = "maplesyrup"
```
The sprite for the maple syrup shows the syrup inside the bottle, and it will look jarring if it stays the same even if the bottle is empty. Might want to add some code that checks how much syrup is left and then updates the sprite accordingly.

KathyRyals replied:
I got new sprites and will update when I can.

SierraKomodo commented:
Author has stated intent to update the sprites - Marking as draft until that's done

Spookerton commented:
@KathyRyals

KathyRyals commented:
Ah fug - will update icons asap

KathyRyals commented:
Don't have time to do the whole "emptying" sprite change thing, you can merge it as is or close it, as you wish.

Spookerton reviewed (approved):
Sure. Someone else can do the icon swap easily enough.
